client/my-sites/plans/current-plan/index.jsx

```diff
@@ -56,12 +51,14 @@ class CurrentPlan extends Component {
 		path: PropTypes.string.isRequired,
 		domains: PropTypes.array,
 		currentPlan: PropTypes.object,
-		isExpiring: PropTypes.bool,
 		requestThankYou: PropTypes.bool,
 		shouldShowDomainWarnings: PropTypes.bool,
 		hasDomainsLoaded: PropTypes.bool,
 		showJetpackChecklist: PropTypes.bool,
 		showThankYou: PropTypes.bool,
+
```

Comment:
Seems like we're also missing the `purchase` proptype here.

Reply:
It actually turns out we're not using `purchase` here anymore. I've removed it in the [latest commit](https://github.com/Automattic/wp-calypso/pull/37642/commits/44386bb5ecdf76bdd7261df0bffafa00c377b886).

Reply:
Oh, even better! 😉